ovs/dal/hybrids/albanode.py

```diff
@@ -136,13 +136,13 @@ def _move(info):
                     # In that case, let's connect to the OSD to see whether we get some info from it
                     try:
                         if osd_id in model_osds:
-                            host = model_osds[osd_id].ips[0]
+                            ip = model_osds[osd_id].ips[0]
                             port = model_osds[osd_id].port
                         else:
                             # @todo check impact once https://github.com/openvstorage/alba/issues/773 lands
-                            host = osd['hosts'][0] if 'hosts' in osd else osd['ips'][0]
+                            ip = osd['ips'][0]
```

Comment:
We should iterate over all ips in case the first ip is not available to get the claimed_by info.
Should be like: try: claimed_by = ... except AlbaException: try with other ip